### DISCUSSION TITLE
Base64 Variants support

### DISCUSSION THREAD

**Discussion by tadpol** [duckdb/duckdb#7052] [Ideas]
Originally I thought I found a bug with the `from_base64()` function, but upon researching it it looks more like a feature request.

I found this https://en.wikipedia.org/wiki/Base64#Variants_summary_table to be really helpful understanding the variants.

It appears that the implementation here is from RFC 4648, which uses `+`, and `/`.  And the data I have uses the URL and filename variation, which uses `-` and `_`.  

So part of this is a feature request to support the variants.

The other part here is a workaround.  I've found, for my data at least, adding a pair of `replace()`s gets things working.

```sql
select from_base64(replace(replace(blob_value,'-','+'),'_','/')) from archive;
```

**Comment by matt-allan** [comment]
Not quite the same thing, but I would love support for variants using other alphabets such as base58, base62, etc.
